app/src/main/java/com/jerboa/ui/components/home/HomePane.kt

```diff
@@ -96,8 +98,10 @@ fun HomeScreen(
     postActionBarMode: PostActionBarMode,
     swipeToActionPreset: SwipeToActionPreset,
     padding: PaddingValues,
+    onPostClick: (PostView) -> Unit,
```

Comment:
We need to highjack this action, because the list-detail view uses its own pseudo navigation.